distribution/changelog.txt

```diff
@@ -40,6 +40,7 @@
 - Feature: [#12885] Add SmallSceneryElement.quadrant to the plugin API.
 - Feature: [#12886] Make all scenery placement and remove actions available to the plugin API.
 - Feature: [#2350, #12922] Add snow, heavy snow and blizzard to weather types.
+- Feature: [#13057] Make GameAction flags accessible by plugins.
```

Comment:
This is in the wrong place. You added to the 0.3.1 section, which has already been released.